.gitignore

```diff
@@ -12,3 +12,79 @@ Makefile.in
 /configure
 /libtool
 /stamp-h1
+Documentation/daxctl/asciidoc.conf
```

Comment:
Please split gitignore and makefile changes to different commits

Reply:
Done